### PR TITLE
Templatize pykickstart version in dracut code

### PR DESCRIPTION
**Pull request by VladimirSlavik**
This is missing from templating to make a branching switch automatic.

(Probably among other things...)

Fixed thanks to @poncovka in https://github.com/rhinstaller/anaconda/pull/4530#pullrequestreview-1277125546

### CODE REVIEW

**Comment by VladimirSlavik** [comment]
The variant where the script does `from pyanaconda.core.kickstart.version import VERSION` would add lots more to initramfs (need a file = add whole dir), so that's not desirable.

I call for a bikeshed session on the new file name 🎉

**Review by rvykydal — APPROVED** [review]
Looks good to me, thank you.

File: dracut/module-setup.sh [hunk 1]
```diff
@@ -58,6 +58,7 @@ install() {
     inst "$moddir/fetch-kickstart-disk" "/sbin/fetch-kickstart-disk"
     inst "$moddir/fetch-updates-disk" "/sbin/fetch-updates-disk"
     inst "$moddir/parse-kickstart" "/sbin/parse-kickstart"
+    inst "$moddir/version.py" "/sbin/version.py"
```
Comment (poncovka) [inline dracut/module-setup.sh:61]: I am not sure if you need to do this. We don't run this code as a script. It needs to be installed as a Python library. See the end of this file.

File: dracut/parse-kickstart [hunk 2]
```diff
@@ -41,7 +41,7 @@ from pykickstart.constants import *
 from collections import OrderedDict
 
 # Import the kickstart version.
-from pykickstart.version import DEVEL as VERSION
+from version import VERSION
```
Comment (poncovka) [inline dracut/parse-kickstart:44]: I guess it could be at least `kickstart_version` to avoid possible future conflicts.

**Review by poncovka — APPROVED** [review]
Otherwise, it looks good to me.

File: dracut/module-setup.sh (hunk 1 above)
Comment (VladimirSlavik) [reply dracut/module-setup.sh]: Let's see how you like the one-liner needed to get a suitable directory :D

**Pull request merged**